src/log.c

```diff
@@ -32,23 +32,23 @@ void log_set_level_from_string(const char *level)
         if (!level)
                 return;
 
-        if (g_ascii_strcasecmp(level, "critical") == 0)
+        if (STR_EQ(level, "critical"))
```

Comment:
This breaks the case insensitive comparison, if I call `log_set_level_from_string("CRITICAL")` it'll fail for example. I'm not sure if we depend on that, but it's good to have.

Reply:
Well, we didn't depend on it, but it makes perfect sense using italics for the loglevel.